<launch>
  <param name="door_handle_detector/handle_distance_door_min_threshold" value="0.05" />
  <param name="door_handle_detector/handle_distance_door_max_threshold" value="0.20" />
  <param name="door_handle_detector/handle_height_threshold" value="0.05" />
  <param name="door_handle_detector/door_min_height" value="1.4" />
  <param name="door_handle_detector/door_max_height" value="3.0" />
  <param name="door_handle_detector/door_min_width" value="0.85" />
  <param name="door_handle_detector/door_max_width" value="1.2" />
  <param name="door_handle_detector/clusters_growing_tolerance" value="0.04" />
  <param name="door_handle_detector/normal_eps_angle_" value="8.0" />
  <node pkg="door_handle_detector" type="door_handle_detector_node" name="door_handle_detector" respawn="true" output="screen">
  <remap from="full_cloud" to="snapshot_cloud" />
  </node>
</launch>
